_accounts.md

```diff
@@ -111,3 +111,50 @@ Retrieves the details about a specific account.
 ### Response
 
 Returns a fully formed [Account Object](#account-object) representing the requested account.
+
+## Adding credit/debit card accounts
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox. These can be used to add card accounts to an Uphold wallet which will then allow for the testing of transactional flows.
+
+<aside class="notice">
+  Any value entered for <i>expiry date</i> and <i>CVV</i> will be accepted as valid when used in conjunction with the card data listed in this section.
+</aside>
+
+### Approved cards
+
+The following is a list of valid card data that can be used to add a card account to an Uphold wallet for testing purposes.
+
+Number           | Brand      | Type   | Country | OCT settlement |
+---------------- | ---------- | ------ | ------- | -------------- |
+5355223761921186 | mastercard | debit  | GB      | instant        |
+5573606426146833 | mastercard | debit  | GB      | instant        |
+5318773012490080 | mastercard | debit  | US      | instant        |
+4921817844445119 | visa       | debit  | GB      | instant        |
+4242424242424242 | visa       | debit  | GB      | instant        |
+4659105569051157 | visa       | debit  | GB      | instant        |
+4024764449971519 | visa       | debit  | US      | instant        |
+5355224542121849 | mastercard | debit  | GB      | N/A            |
+5574357535453624 | mastercard | debit  | GB      | N/A            |
+5573606426146833 | mastercard | debit  | GB      | N/A            |
+5355223761921186 | mastercard | debit  | GB      | N/A            |
+5436031030606378 | mastercard | credit | MU      | N/A            |
+4447336775378848 | mastercard | debit  | US      | N/A            |
```

Comment:
Duplicated. Also asking on the other PR to update.

Reply:
Thanks, we'll hold on a notification that that update has occurred before making anymore changes here.

Reply:
Same as here: https://github.com/uphold/docs/pull/241#discussion_r1455579374.